### PR TITLE
add variable for config_file_mode und config_folder_mode

### PR DESCRIPTION
**Pull request by SimonHoenscheid**

### CODE REVIEW

**Comment by SimonHoenscheid** [comment]
Duplicate of #69

**Comment by SimonHoenscheid** [comment]
@bastelfreak please have a look

File: manifests/init.pp [hunk 1]
```diff
@@ -95,7 +101,9 @@
   String  $config_file            = $telegraf::params::config_file,
   String  $config_file_owner      = $telegraf::params::config_file_owner,
   String  $config_file_group      = $telegraf::params::config_file_group,
+  String  $config_file_mode       = $telegraf::params::config_file_mode,
```
Comment (bastelfreak) [inline manifests/init.pp]: can you please use the filemode datatype? https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/filemode.pp

File: manifests/init.pp [hunk 2]
```diff
@@ -95,7 +101,9 @@
...
   String  $config_folder          = $telegraf::params::config_folder,
+  String  $config_folder_mode     = $telegraf::params::config_folder_mode,
```
Comment (bastelfreak) [inline manifests/init.pp]: same here. https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/filemode.pp. Also this type requires puppetlabs-stdlib 5.0.0 or newer. Please check that the metadata.json includes this version and not an older one.

File: manifests/init.pp (hunk 1 above)
Comment (SimonHoenscheid) [reply manifests/init.pp]: updated

File: manifests/init.pp (hunk 2 above)
Comment (SimonHoenscheid) [reply manifests/init.pp]: updated

File: manifests/init.pp [hunk 3]
```diff
@@ -95,7 +101,9 @@
   String  $config_file            = $telegraf::params::config_file,
   String  $config_file_owner      = $telegraf::params::config_file_owner,
   String  $config_file_group      = $telegraf::params::config_file_group,
+  Filemode  $config_file_mode     = $telegraf::params::config_file_mode,
```
Comment (bastelfreak) [inline manifests/init.pp]: The datatype is namespaced. Please update it like this:
```suggestion
  Stdlib::Filemode  $config_file_mode = $telegraf::params::config_file_mode,
```

Please ensure that the `=` are all aligned.

**Comment by bastelfreak** [comment]
Thanks!

**Pull request merged**